File: Employees.Registration.Gui/EmployeeRegistrationGui/src/app/employee/employeeModel.ts
import { stripSummaryForJitNameSuffix } from '@angular/compiler/src/aot/util'

export class EmployeeModel {
    id?: number 
    name?: string 
    dateOfBirth?: Date
    qualificationId?: string
    experience?: number
    joiningDate?: Date
    salary?: number
    hobbyies?:string 
    qualification?: string 
    designation?: string 
    constructor() { 
        this.hobbyies ="";
    }
}

export class HobbyModel {
    id?: number 
    name?: string   
    status?:boolean
}
export class Qualification {
    id?: number 
    name?: string  
}
export class UserModel {
    email?: string 
    password?: string  
}

File: Employees.Registration/Employees.Registration/Models/EmployeeModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Employees.Registration.Models
{
    public class EmployeeModel
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public string QualificationId { get; set; }
        public int? Experience { get; set; }
        public DateTime? JoiningDate { get; set; }

        public int? Salary { get; set; }
        public string Designation { get; set; }
        public string Hobbyies { get; set; }
        public string Qualification { get; set; }

    }
}
